modular_skyrat/modules/synths/code/surgery/mechanic_steps.dm

```diff
@@ -93,11 +112,15 @@
 
 /datum/surgery_step/add_plating/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/obj/item/stack/sheet/iron/plat = tool
-	if(plat && !(plat.get_amount()<ironamount)) //failproof
+	if(plat && !(plat.get_amount() < ironamount)) //failproof
 		plat.use(ironamount)
 	return TRUE
```

Comment:
So it returns `TRUE` even if it didn't have enough iron? I don't get it. It should fail if that condition isn't met.

Reply:
I didn't even spot that, that's so dumb.